backend/kernelCI_app/views/treeCommitsHistory.py

```diff
@@ -218,8 +220,13 @@ def _process_rows(self, rows: Dict) -> None:
         sanitized_rows = self.sanitize_rows(rows)
 
         for row in sanitized_rows:
+            hardware_filter = row["hardware_compatibles"]
+            if hardware_filter is None or hardware_filter == UNKNOWN_STRING:
```

Comment:
the sanitized rows are simply the original rows but as an object, the `row["hardware_compatibles"]` hasn't been sanitized to receive `UNKNOWN_STRING` yet. In other words, the comparison `hardware_filter == UNKNOWN_STRING` will never be true unless the original row in the database is literally "Unknown", so I think this comparison is not needed